app/api/v2/models/meetup_models.py

```diff
@@ -47,3 +47,14 @@ def get_all_meetups(self):
             "upcoming_meetup": response
         }
         return payload
+
+    def delete_a_specific_meetup(self, id):
+        """Admin can delete a meetup"""
+        fetch_query_for_delete = """SELECT * FROM meetups
+        WHERE meetupId={}""".format(id)
+        response = QuestionerDb.retrieve_one(fetch_query_for_delete)
+        print("Delete this", response)
```

Comment:
What's the purpose of this line?

Reply:
To query if the meetup first exists. So that I can go ahead and delete it

Reply:
If the meetup doesn't exist, returns a false which inform the admin that no meetup of the id being queried is available